src/microbe_stage/Microbe.Contact.cs

```diff
@@ -235,6 +189,21 @@ public float EngulfSize
         }
     }
 
+    /// <summary>
+    ///   Returns true if this microbe or the colony this microbe is part of has the capability to engulf.
+    /// </summary>
+    [JsonIgnore]
+    public bool HasEngulfCapability
```

Comment:
I think this *really* needs a different name, I immediately confused this with `CheckEngulfCapability` while reading the changes here. Maybe something like `MakesSenseToAllowEngulfMode` or? something like that that implies that this cell doesn't necessary have the ability to engulf but still it makes sense for it to try to do so.

Reply:
Renamed.